react/ProductKit.js

```diff
@@ -97,57 +102,56 @@ class ProductKit extends Component {
       delete newProduct.sku.images
       delete newProduct.items
     }
-    
+
     return { ...newProduct, discount, minQuantity }
   }
 
   render() {
     const {
-      productQuery: { product },
+      productQuery: { product, loading },
       showListPrice,
       showLabels,
       showInstallments,
       showBadge,
       badgeText,
     } = this.props
 
-    if (!product.benefits.length) {
+    if (loading || !product.benefits || product.benefits.length === 0) {
```

Comment:
Use `!product.benefits.length` for simplicity.

Reply:
👍